rando_modules/modify_entrances.py

```diff
@@ -883,12 +905,16 @@ def get_glitched_logic(
     # Mt. Rugged
     if glitch_settings.mt_rugged_quake_hammer_and_letter_with_laki:
         all_new_edges.extend(edges_iwa_add_quake_hammer_and_letter_laki)
+    if glitch_settings.mt_rugged_quake_hammer_room_jumpless_clippy:
+        all_new_edges.extend(edges_iwa_add_quake_hammer_room_jumpless_clippy)
     if glitch_settings.parakarryless_mt_rugged_seed:
         all_new_edges.extend(edges_iwa_add_parakarryless_mt_rugged_seed)
     if glitch_settings.parakarryless_mt_rugged_seed_clippy:
         all_new_edges.extend(edges_iwa_add_parakarryless_mt_rugged_seed_clippy)
     if glitch_settings.parakarryless_mt_rugged_star_piece:
         all_new_edges.extend(edges_iwa_add_parakarryless_star_piece_laki)
+    if glitch_settings.parakarryless_mt_rugged_star_piece_clippy:
+        all_new_edges.extend(edges_iwa_add_parakarryless_star_piece_clippy)
```

Comment:
`edges_iwa_add_parakarryless_star_piece_clippy` is not defined (Pylance(reportUndefinedVariable))